spec/test_controllers/test_post_controller.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: 'request' do
+  describe 'GET posts#index' do
+    before :each do
+      user = User.create(name: 'name for test')
+      user.posts.create(title: 'Post test', text: 'This is for testing')
+      get "/users/#{user.id}/posts"
+    end
+
+    it 'is success ' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'renders index template with right text' do
+      expect(response.body).to match('Posts & Comments')
+      expect(response.body).to match('name for test')
+    end
+
+    it 'does not render a different template' do
+      expect(response).to_not render_template(:show)
+    end
+  end
+
+  describe 'GET posts#show' do
+    before :each do
+      user = User.create(name: 'test')
+      post = Post.create(author_id: user.id, title: 'Post test', text: 'This is for testing')
+      get "/users/#{user.id}/posts/#{post.id}"
+    end
+
+    it 'is success ' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'renders "show template" with right text' do
+      expect(response.body).to match('Post test')
+      expect(response.body).to match('Show Comments')
+    end
+
+    it 'does not render a different template' do
+      expect(response).to_not render_template(:index)
+    end
+  end
```

Comment:
- [ ] I am not able to run `spec` I suggest that you change the name of the folder from `test_controllers` to `spec/requests/`
<img width="1440" alt="Screenshot 2023-10-04 at 10 33 53" src="https://github.com/Aqurds/ror-blog-app/assets/44745855/6dd8a739-a467-4d52-b7f2-9993c93e3546">

Reply:
- [ ] Change names of the folder from `test_post_controller.rb` to `posts_spec.rb` and `users_spec.rb`

Reply:
Changes implemented.